mwpc/mwpc3/R3BMwpc3Cal2Hit.cxx

```diff
@@ -157,7 +157,7 @@ void R3BMwpc3Cal2Hit::S467()
         Int_t padId = 0;
         Int_t padmx = -1, padmy = -1;
         Double_t q = 0., qmx = 0., qmy = 0., qleft = 0., qright = 0., qdown = 0., qup = 0.;
-        Double_t x = -1000., y = -1000.;
```

Comment:
This change is not needed because you don't need the mwpc3, right?

Reply:
Yes, for this experiment but I would like to apply this anyway for the offline analysis in the previous experiments. Is it difficult to solve the clang-tidy issues?

Reply:
I can accept it directly because during the compilation of r3broot this class is only used if the sofia repo is added.